airgun/views/subscription.py

```diff
@@ -89,12 +134,14 @@ class SubscriptionListView(BaseLoggedInView, SubscriptionSearchableViewMixin):
             'Name': Text("./a"),
         }
     )
+
     add_button = Button(href='subscriptions/add')
     manage_manifest_button = Button('Manage Manifest')
     export_csv_button = Button('Export CSV')
     delete_button = Button('Delete')
     progressbar = ProgressBar('//div[contains(@class,"progress-bar-striped")]')
     confirm_deletion = DeleteSubscriptionConfirmationDialog()
+    columns_filter_checkboxes = SubscriptionColumnsFilter(".//form[div[contains(@class, 'filter')]]/div/i")
```

Comment:
Personally it would make sense for me to add it above, simply because this is the first element on the page you will see going left-right, top-bottom.

But usually we don't really care about order of widgets, so you can leave it as is.